src/lib/collab/channel.js

```diff
@@ -60,13 +75,23 @@ export class Channel {
     this.queuedStep = undefined
   }
 
+  /**
+   * Ensure all local steps are sent to the server
+   */
+  async ensureEmptyQueue() {
+    return this.waitForEmptyQueue
```

Comment:
~~Are we sure we do not want to call the function here ?~~

I think this.waitForEmptyQueue should be renamed, it seems like it is a method when it is a promise. We should avoid using verbs for non function variables.

Reply:
-> emptyQueuePromise